Select/Select.d.ts

```diff
@@ -0,0 +1,32 @@
+import MaterialComponent from '../MaterialComponent';
+import { VNode } from 'preact';
+import { MDCFoundation, MDCComponent } from '../MaterialComponentsWeb';
+import { Item as ListItem } from '../List/List';
+
+export interface ISelectProps extends JSX.HTMLAttributes {
+  disabled?: boolean;
```

Comment:
Missing
`basic`
`onChange`
`selectedIndex`

Reply:
Also fixed but apparently Github didn't register it.